bin2c/bin2c.c

```diff
@@ -44,19 +47,34 @@
     }
 #endif
 
+	unsigned char * fprintList(FILE *outputFile, unsigned char *bufptr, unsigned char len, const char* formatStr)
+	{
+		for (unsigned char j = 0; j < len; j++)
+		{
+			fprintf(outputFile, formatStr, *(bufptr++));
+			fprintf(outputFile, ",");
+		}
+		fprintf(outputFile, "\n");
+		return bufptr;
+	}
+
 int main(int argc, char *argv[])
 {
     FILE          *inputFile = NULL;
     FILE          *outputFile = NULL;
     void          *buffer;
-    unsigned int  fileLen, i, j, major, minor;
+    unsigned int  fileLen, i, major, minor;
```

Comment:
Prefer one declaration per line - as above with the FILE*'s.

Reply:
No problem, I will split into one declaration per line.